_services/automation.html

```diff
@@ -0,0 +1,100 @@
+      <li>Marketing Automation</li>
+      <li>Customer Success Optimization</li>
+      <li>CRM Automation</li>
+    </ul>
+    <p>
+      Our automation services focus on key components of your tech stack, and
+      getting them to work together through the use of Automation tools such as
+      Customer Relationship Management (CRM) Systems Pipedrive and HubSpot,
+      Marketing Automation Platforms like HubSpot Marketing Hub, Sales
+      Automation Tools like Pipedrive’s Sales Automation System, Workflow
+      Automation Platforms like Zapier, Email and Communication Automation tools
+      like Pipedrive’s Campaigns feature and HubSpot’s Workflow Automations, and
+      more.
+    </p>
+    <p>
+      By leveraging these advanced technologies we streamline your business
+      processes, reduce manual tasks, and drive sustainable growth—saving you
+      valuable time and resources.
+    </p>
+  </section>
+  <img class="side-image" src="/img/meeting-room-original.jpg" />
+</div>
+
+<div class="flex-row">
+  <img class="side-image" src="/img/business-man-away.jpg" />
+  <section>
+    <div class="h2-wrapper">
+      <img src="/img/explore-icon.svg" class="h2-icon" />
+      <h2>Why choose us</h2>
+    </div>
+    <p>
+      We work directly with you and your teams to understand your businesses
+      unique goals and identify opportunities for automation that can streamline
+      operations and drive growth.
+    </p>
+    <p>
+      We have run, and automated, our own successful companies - we know
+      firsthand the common pitfalls businesses encounter. Our expertise allows
+      us to help you avoid those obstacles through smart automation strategies.
+    </p>
+    <p>
+      We have certified technical expertise in automation platforms such as
+      <a href="/hubspot">HubSpot</a>, <a href="/pipedrive">Pipedrive</a>, and
+      other leading systems. Through these partnerships, we implement custom
+      automation solutions tailored to your business needs
+    </p>
+    <p>
+      We specialise in working with startups and scaleups and understand the
+      importance of early growth stages. We’ll help you build a scalable
+      automation framework that ensures long-term success and sustainable
+      growth.
+    </p>
+    <p>
+      Plug and Play Expertise. We hit the ground running, quickly integrating
+      into new companies and optimizing business processes through automation.
+      Our ability to automate and streamline processes helps you achieve results
+      fast.
+    </p>
+    <p>
+      Partnering with us means turning your revenue goals into efficient,
+      automated workflows. Our team will swiftly implement these strategies,
+      streamlining your operations through automation.
+    </p>
+    <p>
+      We also offer <a href="/">DevOps services</a> so should you require
+      technical leadership, technical counseling or pair programming and
+      coaching we are your dev team.
```

Comment:
I would framework the devops need for the type of customer reading this page as: once you have set up standard software to get the low hanging fruit, and are ready to build the systems that will give you a fully unique advantage in your industry, for the parts of the business that give you your advantage in the market, and thus where conforming to standard software would make you lose your edge.